contracts/crypto-verify/src/msg.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(clippy::field_reassign_with_default)] // see https://github.com/CosmWasm/cosmwasm/issues/685
+
+use cosmwasm_std::{Binary, Deps, StdResult};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InitMsg {}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum HandleMsg {
+    VerifySignature {
```

Comment:
Why is this a HandleMsg not a QueryMsg?

Reply:
Also, maybe `VerifyCosmosSignature` in case you add an Eth or Tendermint (ed25519) demo to this contract as well

Reply:
> Why is this a HandleMsg not a QueryMsg?

Yeah, I had the same question... will refactor.

> Also, maybe VerifyCosmosSignature in case you add an Eth or Tendermint (ed25519) demo to this contract as well

Yes. Will do that refactoring when adding the ed25519 endpoint